mappings/net/minecraft/unmapped/C_mmhxrdox.mapping

```diff
@@ -1,7 +1,5 @@
 CLASS net/minecraft/unmapped/C_mmhxrdox net/minecraft/unmapped/C_mmhxrdox
-	COMMENT A class holding unknown constants.
-	COMMENT
-	COMMENT @apiNote From the package structure, this class appears to be world-related,
-	COMMENT but it seems to be the exclusive class in a package. Its neighboring packages
-	COMMENT are about portal teleporting and persistent state, yet these constants don't
-	COMMENT seem to relate to those.
+	COMMENT Holds some redstone signal constants.
```

Comment:
Good. Was wondering when we would get around to this

Reply:
I still don't know in which package to put it :pensive:

Reply:
Lets put it in the world package

Reply:
Sounds cursed to me and a bit mojmap-y but, eh, why not.

Reply:
It's that or block, which doesn't feel quite right either